community/contribute/elasticsearch-develop.md

```diff
@@ -1,6 +1,7 @@
 ---
+id: elasticsearch-develop
```

Comment:
No we don't have to but thanks to that we can adapt slug URL `{host}/community/contribute/{pageId}` and also we can relay on it during some connection with configuration so I recommend to leave it as it is.

Reply:
🤔 I think we quite often rename the files, so that maybe it makes sense to have new paths and then use redirect as here: https://github.com/capactio/website/blob/main/redirects.js.

But yeah, let's keep it as it is for now 👍